### ISSUE TITLE
Recording shows chrome welcome page

### ISSUE THREAD

**Issue by mandriv** [OpenVidu/openvidu#548]
**Describe the bug**
The created recording contains only Chrome welcome page (Welcome to Chrome followed by Web browsing with Google smarts).

I am using a recording feature in `COMPOSED` mode with audio and video enable. Layout is `BEST_FIT`. 
The deployment is done via CloudFormation on AWS. I tried using different instances as well as different versions (2.14 and 2.15). The deployment configuration is a follows (.env file with commented out values trimmed):
```
DOMAIN_OR_PUBLIC_IP=live.vizcareer.com
OPENVIDU_SECRET=<redacted>
CERTIFICATE_TYPE=letsencrypt
LETSENCRYPT_EMAIL=wojciech.cichoradzki@gmail.com
HTTPS_PORT=443
OPENVIDU_RECORDING=true
OPENVIDU_RECORDING_PATH=/opt/openvidu/recordings
OPENVIDU_RECORDING_CUSTOM_LAYOUT=/opt/openvidu/custom-layout
OPENVIDU_RECORDING_PUBLIC_ACCESS=true
OPENVIDU_RECORDING_NOTIFICATION=moderator
OPENVIDU_RECORDING_AUTOSTOP_TIMEOUT=120
OPENVIDU_STREAMS_VIDEO_MAX_RECV_BANDWIDTH=1000
OPENVIDU_STREAMS_VIDEO_MIN_RECV_BANDWIDTH=300
OPENVIDU_STREAMS_VIDEO_MAX_SEND_BANDWIDTH=1000
OPENVIDU_STREAMS_VIDEO_MIN_SEND_BANDWIDTH=300
OPENVIDU_WEBHOOK=false
OPENVIDU_WEBHOOK_EVENTS[sessionCreated,sessionDestroyed,participantJoined,participantLeft,webrtcConnectionCreated,webrtcConnectionDestroyed,recordingStatusChanged,filterEventD
ispatched,mediaNodeStatusChanged]
OPENVIDU_SESSIONS_GARBAGE_INTERVAL=900
OPENVIDU_SESSIONS_GARBAGE_THRESHOLD=3600
OPENVIDU_CDR=false
OPENVIDU_CDR_PATH=/opt/openvidu/cdr
```
I don't have the same issue when running local docker image
I am accessing video via public link correctly returned to me after stopRecording method is called on my node server.
I also don't get any error messages in openvidu logs.
**Expected behavior**
The created recording of a session is being displayed.
**Wrong current behavior**
The created recording contains only Chrome welcome page (Welcome to Chrome followed by Web browsing with Google smarts).

**OpenVidu tutorial where to replicate the error**
Official tutorials with local deployments are fine, I am unable to get the recording only on AWS.

**OpenVidu deployment info**
- AWS deployment as explained in OpenVidu Docs (https://docs.openvidu.io/en/stable/deployment/deploying-aws/)

**Client device info (if applicable)**
- Firefox Dev 62.2.b macOS Mojave

**Screenshots**
Video is available here: https://streamable.com/75vgeg

**Additional context**
Add any other context about the problem here. For example, attach any useful logs related to the issue.

**Comment by pabloFuente** [comment]
Our tests indicate COMPOSED recording is working fine in AWS deployments of OpenVidu 2.15.0. Can you paste here the initial log of OpenVidu Server? I mean from these lines:

```
openvidu-server_1  | ______________________________________________
openvidu-server_1  |    ____               __      ___     _       
openvidu-server_1  |   / __ \              \ \    / (_)   | |      
openvidu-server_1  |  | |  | |_ __   ___ _ _\ \  / / _  __| |_   _
openvidu-server_1  |  | |  | | '_ \ / _ \ '_ \ \/ / | |/ _` | | | |
openvidu-server_1  |  | |__| | |_) |  __/ | | \  /  | | (_| | |_| |
openvidu-server_1  |   \____/| .__/ \___|_| |_|\/   |_|\__,_|\__,_|
openvidu-server_1  |         | |
openvidu-server_1  |         |_|             version 2.15.0
openvidu-server_1  | ______________________________________________                                          
openvidu-server_1  | 
```

To these ones:

```
openvidu-server_1  | ----------------------------------------------------
openvidu-server_1  | 
openvidu-server_1  |    OpenVidu is ready!
openvidu-server_1  |    ---------------------------
openvidu-server_1  | 
openvidu-server_1  |    * OpenVidu Server: https://your-ip/
openvidu-server_1  | 
openvidu-server_1  |    * OpenVidu Dashboard: https://your-ip/dashboard/
openvidu-server_1  | 
openvidu-server_1  | ----------------------------------------------------

```

**Comment by pabloFuente** [comment]
Maybe this is related to this problem: https://github.com/OpenVidu/openvidu/issues/541
The outcome is the same, at least. Take a look at that and tell us if it's the same problem.

**Comment by mandriv** [comment]
Hi @pabloFuente, thank you so much, the issue described in #541 was the root cause, I had `#` and `^` characters in my secret string. Once I changed it to only letters [A-z] the recordings started to work - I agree with suggestion to add a note to docs about this known issue until it's resolved.
Another issue I encountered was about the quality of recordings I started receiving - they were dropping frames and sometimes going full black. If anyone has the same issue on AWS - I upgraded my instance to c5.xlarge per suggestions in this article https://medium.com/@openvidu/openvidu-load-testing-a-systematic-study-of-openvidu-platform-performance-b1aa3c475ba9 and everything was great, I get stable composed recordings in HD quality.

Now everything is working perfectly, thank you for reaching out and thank you for the amazing piece of software.

I'm going to close this issue.
